signer/api/api.go

```diff
@@ -166,24 +167,23 @@ func Sign(sigServices signer.SigningServiceIndex) http.Handler {
 			return
 		}
 
-		signer, err := sigService.Signer(sigRequest.KeyID)
-		if err == keys.ErrInvalidKeyID {
-			w.WriteHeader(http.StatusNotFound)
-			w.Write([]byte(err.Error()))
-			return
-		} else if err != nil {
-			// We got an unexpected error
+		tufKey := cryptoService.GetKey(sigRequest.KeyID.ID)
```

Comment:
Good point. This was the result of a poor translation of older code. I just fixed this both here and in rpc_api.go.